selfdrive/car/toyota/carcontroller.py

```diff
@@ -49,15 +61,15 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, hud_alert,
     # *** compute control surfaces ***
 
     # gas and brake
+    apply_gas = 0.
+    apply_accel = actuators.gas - actuators.brake
 
-    apply_gas = clip(actuators.gas, 0., 1.)
-
-    if CS.CP.enableGasInterceptor:
-      # send only negative accel if interceptor is detected. otherwise, send the regular value
+    if CS.CP.enableGasInterceptor and enabled and CS.out.vEgo < MIN_ACC_SPEED and \
+            apply_accel * CarControllerParams.ACCEL_SCALE > coast_accel(CS.out.vEgo):
       # +0.06 offset to reduce ABS pump usage when OP is engaged
-      apply_accel = 0.06 - actuators.brake
-    else:
-      apply_accel = actuators.gas - actuators.brake
+      # converts desired acceleration to gas percentage for pedal
+      apply_gas = clip(compute_gb_pedal(apply_accel * CarControllerParams.ACCEL_SCALE, CS.out.vEgo), 0., 1.)
+      apply_accel = 0.06
```

Comment:
Do we need to send 0.06 * 3? Or could it be even a higher number? Coast accel under 5 mph is about 0.5 m/s/s so it could brake when we're sending 0% gas. Setting it above `.5/3` might fix it from braking once the gas pedal is released and we want to truly coast